hack/ukify/main.go

```diff
@@ -233,7 +233,7 @@ func run() error {
 	}{
 		Name:    version.Name,
 		ID:      strings.ToLower(version.Name),
-		Version: version.Tag,
+		Version: version.Tag, // TODO: this is abbrev tag, not the full tag, it should be fixed by building from sources + generate data
```

Comment:
ukify should be built after `generate` step, so that `version.Tag` is not abbreviated ?

Reply:
fixed